paper-dropdown-menu.html

```diff
@@ -390,15 +399,17 @@
          * optional `label` property.
          */
         _selectedItemChanged: function(selectedItem) {
-          var value = '';
+          var value = '',
+              label = '';
           if (!selectedItem) {
-            value = '';
+            value = label = '';
           } else {
-            value = selectedItem.label || selectedItem.textContent.trim();
+            label = selectedItem.label || selectedItem.textContent.trim();
+            value = selectedItem[this.attrForSelected] || label;
```

Comment:
> This is useful since `paper-dropdown-menu` doesn't have any way of setting the selected item. Selection must be done on the `paper-listbox`

Very good point about the selection staing with in the list-element it self :+1:  I'll make the change and update the PR